run_analysis.py

```diff
@@ -11,21 +11,22 @@
 from hepaccelerate.utils import Results, NanoAODDataset, Histogram, choose_backend
 
 import tensorflow as tf
-from keras.models import load_model, model_from_json
+from tensorflow.keras.models import load_model, model_from_json
 import itertools
 from lib_analysis import mse0,mae0,r2_score0,decorr
+from lib_analysis import trijet_feats, dijet_feats, lep_feats
 
 from definitions_analysis import histogram_settings
 
 import lib_analysis
 from lib_analysis import vertex_selection, lepton_selection, jet_selection, load_puhist_target, compute_pu_weights, compute_lepton_weights, compute_btag_weights, chunks, evaluate_DNN, calculate_variable_features
 
-config = tf.ConfigProto()
-config.gpu_options.allow_growth=True
-sess = tf.Session(config=config)
+#config = tf.ConfigProto()
```

Comment:
warum kommentierst du das aus? wenn du in mehreren batchen laufen laesst, kann das zu problemen fuehren (memory leakages)

Reply:
Welches TensorFlow benutzt ihr? Wir benutzen 2.0 und da ist ConfigProto deprecated (stürzt mit Fehler ab). Ich schaue nochmal wie das Äquivalent in 2.0 ist.

Reply:
okay, kann sein, dass das anders ist.. sei einfach vorsichtig, da die cupy funktionen teilweise platz auf der GPU brauchen, die ihnen Tensorflow nicht lassen